src/main/java/frc/robot/commands/auto/AutoRoutines.java

```diff
@@ -33,57 +33,106 @@
 import frc.robot.subsystems.PhotonSubsystem;
 import frc.robot.subsystems.ShooterSubsystem;
 import frc.robot.subsystems.SwerveSubsystem;
+import frc.robot.commands.BlingCommand;
+import frc.robot.commands.BlingCommand.BlingColour;
 
 public class AutoRoutines extends SubsystemBase {
     
     // PathPlannerPath speakerPath = PathPlannerPath.fromPathFile("Speaker Path");
    
-                        
+    PathPlannerAuto //fourNoteAuto,
+                    // twoNoteAuto,
+                    // threeNoteAuto,
+                    //twoNoteLeftAuto,
+                    //twoNoteCenter,
+                    //threeNoteCenterSourceSideNote,
+                    //threeNoteCenterAmpSideNote,
+                    //oneNoteSourceSide,
+                    Right_R_CD,
+                    Right_R_CD_R,
+                    RIGHTCenter_R_CD,
+                    RIGHTCenter_R_CD_R,
+                    LEFTCenter_R_CD,
+                    LEFTCenter_R_CD_R,
+                    Left_R_CD,
+                    Left_R_CD_R,
+                    centerMove;
+                    //twoNoteSourceSide;
     
 
     public AutoRoutines() {
         registerCommandsToPathplanner();
 
+        Right_R_CD = new PathPlannerAuto("rightReefCd");
+        Right_R_CD_R = new PathPlannerAuto("rightReefCdReef");
+        Left_R_CD = new PathPlannerAuto("leftReefCd");
+        Left_R_CD_R = new PathPlannerAuto("leftReefCdReef");
+        RIGHTCenter_R_CD = new PathPlannerAuto("rightCenterReefCd");
+        RIGHTCenter_R_CD_R = new PathPlannerAuto("rightCenterReefCdReef");
+        LEFTCenter_R_CD_R = new PathPlannerAuto("leftCenterReefCd");
+        LEFTCenter_R_CD = new PathPlannerAuto("leftCenterReefCdReef");
+        centerMove = new PathPlannerAuto("centerMove");
+
+         
         // twoNoteAuto = new PathPlannerAuto("twoNoteSpeaker");
         // threeNoteAuto = new PathPlannerAuto("threeNoteSpeaker");
        
     }
 
     public void registerCommandsToPathplanner() {
-        // NamedCommands.registerCommand("MakeIntakeMotorSpin", new SequentialCommandGroup(
-        //     new MakeIntakeMotorSpin(3.0,2), // Move arm to intake setpoint
-        //     new WaitCommand(1)
-        // ));
+         NamedCommands.registerCommand("purpleBling", new BlingCommand(BlingColour.PURPLE));
```

Comment:
Please only keep two or three bling commands. We do not need all the bling colors in Auto.